app/views/base/layout.scala

```diff
@@ -124,6 +124,12 @@ object layout {
       )
     )
 
+  private def switchLanguage(implicit ctx: Context) =
+    spaceless(s"""<form method="post" action="/translation/select" class="header-langs">
+    ${ if (ctx.lang.language == "en") """<button type="submit" name="lang" value="ja-JP" title="ja-JP">日本語</button>""" else "" }
+    ${ if (ctx.lang.language == "ja") """<button type="submit" name="lang" value="en-US" title="en-US">English</button>""" else "" }
```

Comment:
I'm curious how this works, given that I think somewhere else I observed that the project's main language is `en-GB`.

Reply:
Haven't tested but I think both en-GB and en-US both return "en" for `ctx.lang.language`. Should I replace `value="en-US"` with `value="en-GB"`?

Reply:
Oh just realized, now that lishogi automatically uses browser language as default (https://github.com/WandererXII/lishogi/issues/65#issuecomment-734258450) the "English" button should still show up even if the language was, say, French. Will update.